TESTING.md

```diff
@@ -14,33 +14,81 @@ Note that because the logs are from GNOME Shell, there will be messages from all
 
 ## Checklists
 
-Tasks for a tester to verify when approving a patch
+Tasks for a tester to verify when approving a patch. Use complex window layouts and at least two displays. Turn on active hint during testing.
+
+## With tiling enabled
+
+### Tiling
+
+- [ ] Super direction keys changes focus in the correct direction
+- [ ] Windows moved with the keyboard tile into place
+- [ ] Windows moved with the mouse tile into place
+- [ ] Windows swap with the keyboard (test with different size windows)
+- [ ] Windows can be resized with the keyboard (Test resizing four windows above, below, right, and left to ensure shortcut consistency)
+- [ ] Windows can be resized with the mouse
+- [ ] Minimizing a window detaches it from the tree and re-tiles remaining windows
+- [ ] Unminimizing a window re-tiles it into its previous location
+- [ ] Maximizing removes the active hint and covers tiled windows
+- [ ] Umaximizing adds active hint and re-tiles into place
```

Comment:
This currently doesn't always re-tile the window into the space it was formerly in. Should we add `(Not required, currently has a bug)`?